{{cookiecutter.project_shortname}}/Pipfile

```
@@ -0,0 +1,20 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+check-manifest = ">=0.25"
+
+[packages]
+invenio-app-ils = {extras = ["postgresql", "elasticsearch7", "lorem"], version = "~=1.0.0a67"}
```

we will release `rc1`
```suggestion
invenio-app-ils = {extras = ["postgresql", "elasticsearch7", "lorem"], version = "~=1.0.0rc1"}
```